packages/blockchain-wallet-v4-frontend/package.json

```diff
@@ -71,7 +71,8 @@
       "<rootDir>/../../node_modules/enzyme-to-json/serializer"
     ],
     "transform": {
-      "^.+\\.(js|jsx)$": "babel-jest"
+      "^.+\\.(js|jsx)$": "babel-jest",
+      "^.+\\.tsx?$": "ts-jest"
```

Comment:
We should also add the .ts extension here, typically files which don't contain any JSX should be .ts files, and not .tsx

Reply:
It should handle both 👍